src/components/location-selection-inputs/LocationSelectionInputs.tsx

```diff
@@ -1,61 +1,61 @@
-import { SyntheticEvent, useCallback, useEffect, useMemo } from 'react'
+import {
+  type FilterOptionsState,
+  type SxProps,
+  createFilterOptions
+} from '@mui/material'
+import { type SyntheticEvent, useCallback, useMemo } from 'react'
```

Comment:
```suggestion
import { useCallback, useMemo } from 'react'
```
If you've decided to refactor the component, write the `React.SyntheticEvent` type when it's needed instead.